app/Http/Controllers/FineUploaderController.php

```diff
@@ -15,13 +15,13 @@
 
 class FineUploaderController extends Controller
 {
-    public function endpoint(Request $request){
+    public function endpoint(Request $request) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->